docs/advanced-usage.md

```diff
@@ -418,3 +418,30 @@ Please refer to the [Ensuring workflow access to your package - Configuring a pa
 
 ### always-auth input
 The always-auth input sets `always-auth=true` in .npmrc file. With this option set [npm](https://docs.npmjs.com/cli/v6/using-npm/config#always-auth)/yarn sends the authentication credentials when making a request to the registries.
+
+## Enabling Corepack
```

Comment:
```suggestion
## corepack input
**Experimental: This feature is marked as experimental in Node versions 18-22. It will only ever be supported for node versions that support it.**
```